pom.xml

```diff
@@ -1,4 +1,4 @@
-<project xmlns="http//maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ <project xmlns="http//maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
@sherrif10 thanks for working on this,was this change important?

Reply:
Thats a minor change